log/uds.go

```diff
@@ -111,6 +115,8 @@ func (u *udsCore) Write(entry zapcore.Entry, fields []zapcore.Field) error {
 	if err != nil {
 		return fmt.Errorf("failed to write log to uds logger: %v", err)
 	}
+	u.mu.Lock()
+	defer u.mu.Unlock()
```

Comment:
Not: no advantage in using defer instead of unlocking before return.

Reply:
Done.